bin/generate.dart

```diff
@@ -86,23 +94,28 @@ class DiagramGenerator {
   /// targeting.
   String deviceTargetPlatform = '';
 
-  Future<void> generateDiagrams(List<String> categories, List<String> names) async {
+  Future<void> generateDiagrams(
```

Comment:
Were these added line break formatting changes intended, or an auto formatting thing? It looks like an auto formatter might have hit the whole file. If intended, I don't think they make these lines more readable, but YMMV 😏 .

Reply:
It was dartfmt.  Sorry, I must have hit the wrong shortcut in the editor.  I'll revert these.